src/services/workflow.service.js

```diff
@@ -34,29 +27,38 @@ import {
   processMutations,
   tokenise
 } from '@/utils/aotf'
+import store from '@/store/index'
+import { createApolloClient } from '@/graphql/index'
+/* eslint-disable no-unused-vars */
+import { DocumentNode, print } from 'graphql'
+import { SubscriptionClient } from 'subscriptions-transport-ws'
+/* eslint-enable no-unused-vars */
 
-class WorkflowService extends GQuery {
+class WorkflowService {
   /**
    * @constructor
    * @param {string} httpUrl
    * @param {SubscriptionClient|null} subscriptionClient
    */
   constructor (httpUrl, subscriptionClient) {
-    super()
-    this.query = null
```

Comment:
Before the Workflow Service allowed a single query. Every other query would be merged back to this original query. Now it supports multiple queries. You can have 1 merged query (e.g. when you have Dashboard + GScan), or you can have 2 or more (e.g. when you have GScan, and also the other query for Tree/Table views, which uses variables like `{ workflowId : "kinow|five" }`. Note these variables cannot be applied to the GScan query as that would limit the data returned, hence the 2 subs/queries in this example).